types/bootstrap-colorpicker/index.d.ts

```diff
@@ -0,0 +1,275 @@
+     */
+    slidersHorz?: object;
+
+    /**
+     * Customizes the default colorpicker HTML template.
+     */
+    template?: string;
+
+    /**
+     * 	By default, the colorpicker is aligned to the right of the input. If you need to switch it to the left, set align to 'left'.
+     * 
+     * Default: 'right'
+     */
+    align?: ColorPickerAlignOptions;
+
+    /**
+     * Adds this class to the colorpicker widget.
+     * 
+     * Default: null
+     */
+    customClass?: string;
+
+    /**
+     * 	List of pre selected colors (hex format). If you choose one of these colors, the alias is returned instead of the hex code.
+     * 
+     * Default: null
+     */
+    colorSelectors?: object;
+
+    /**
+     * Fallback color string that will be applied when the color failed to be parsed. If null, it will keep the current color if any.
+     * 
+     * Default: null
+     */
+    fallbackColor?: string;
+
+    /**
+     * 	Fallback color format (e.g. when not specified or for alias mode, when selecting non aliased colors)
+     * 
+     * Default: hex
+     */
+    fallbackFormat?: string;
+}
+
+interface Color {
+    colors: {};
+    fallbackFormat: string;
+    fallbackValue: string;
+    hexNumberSignPrefix: boolean;
+    origFormat: string;
+    predefinedColors: {};
+    value: {
+        h: number;
+        s: number;
+        b: number;
+        a: number;
+    };
+
+    /**
+     * Set a new color. The value is parsed and tries to do a quess on the format.
+     */
+    setColor(value: string): void;
+
+    /**
+     * Set the HUE with a value between 0 and 1.
+     */
+    setHue(value: number): void;
+
+    /**
+     * Set the saturation with a value between 0 and 1.
+     */
+    setSaturation(value: number): void;
+
+    /**
+     * Set the brightness with a value between 0 and 1.
+     */
+    setBrightness(value: number): void;
+
+    /**
+     * Set the transparency with a value between 0 and 1.
+     */
+    setAlpha(value: number): void;
+
+    /**
+     * Returns a hash with red, green, blue and alpha.
+     */
+    toRGB(): string;
+
+    /**
+     * Returns a string with HEX format for the current color.
+     */
+    toHex(): string;
+
+    /**
+     * Returns a hash with HSLA values.
+     */
+    toHSL(): string;
+
+    /**
+     * Returns current color as string in specified format.
+     */
+    toString(format: ColorFormat): string;
+}
+
+interface ColorPicker {
+    color: Color;
+    component: boolean;
+    container: boolean;
+    disabled: boolean;
+    element: JQuery;
+    format: string;
+    input: JQuery;
+    options: ColorPickerOptions;
+    picker: JQuery;
+}
+
+interface ColorPickerEventObject extends JQueryEventObject {
+    color: Color;
+}
+
+interface JQuery {
+    /**
+     * Initializes an colorpicker.
+     */
+    colorpicker(): JQuery;
+
+    /**
+     * Initializes an colorpicker.
+     */
+    colorpicker(options: ColorPickerOptions): JQuery;
```

Comment:
Since this and the above method have the same documentation, I would suggest removing the one with no arguments and write this as `colorpicker(options?: ColorPickerOptions): JQuery;`